RightTriangleLibrary/RightTriangleLibrary/RightTriangle.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+
+namespace RightTriangleLibrary
+{
+    
+        public struct RightTriangle
+        {
+            public double SideA;
+            public double SideB;
+            public readonly double Hypotenuse;
```

Comment:
Это неправильно. Гипотенузу нужно было сделать свойством только для чтения, а не полем только для чтения. Как поле, его значение уже не может быть изменено после установки значения поля конструктором. В то же время значения катетов могут быть изменены (а гипотенуза при этом останется незменной!). Нарушится целостность данных.
```suggestion
            public double Hypotenuse
            {
                get
                {
                    return Math.Sqrt(sideA * sideA + sideB * sideB);
                }
            }
```